docs/pages/server-access/guides/openssh.mdx

```diff
@@ -340,6 +340,89 @@ authenticate the host via the certificate we generated earlier.
 
 </Details>
 
+<Details title="Using Trusted Clusters?">
```

Comment:
Looking at this guide now, it seems a bit awkward that the paragraph before the Details boxes begins, "If you are using Trusted Clusters...". Maybe we can remove the "If you are using Trusted Clusters..." paragraph and begin this Details box with something like,

```markdown
If you are using Trusted Clusters, the `tsh config` command will print an OpenSSH
 client configuration block for the root cluster and all currently known leaf clusters.

This configuration block uses a `ProxyCommand` directive to connect to SSH 
servers in a leaf cluster through the root cluster's Proxy Service.
```